.github/workflows/ci.yml

```diff
@@ -215,7 +216,7 @@ jobs:
           && startsWith(matrix.os, 'windows-')
         shell: bash
         run: |
-          choco install -y --limit-output winfsp --version=${{ env.winfsp-version }}
+          choco install -y --limit-output winfsp --version=${{ matrix.winfsp-version }}
```

Comment:
so the commant was executed as `choco install -y --limit-output winfsp --version=` ? funny we didn't get an error from :/

Reply:
Yes totally

I'm also surprise the error take that long to manifest